modules/calcite/src/test/sql/subquery/scalar/test_grouped_correlated_subquery.test_ignore

```diff
@@ -1,8 +1,8 @@
 # name: test/sql/subquery/scalar/test_grouped_correlated_subquery.test
 # description: Test correlated subquery with grouping columns
 # group: [scalar]
-# Ignore https://issues.apache.org/jira/browse/IGNITE-15603
 # Ignore https://issues.apache.org/jira/browse/IGNITE-15607
+# Ignore https://issues.apache.org/jira/browse/IGNITE-16000
```

Comment:
Do we need to add non-working/ignored tests queries?

Reply:
I don't get this comment. The new problem was identified, the new ticket filled, added information about ticket to this file, but no new tests were added to ignored file.